doc/WebSite/MySql-Integration.md

```diff
@@ -0,0 +1,68 @@
+
+### Download Starter Template
+
+I will use a starter template with **ASP.NET Core** and **Entity Framework Core** to integrate MySql. 
+So I downloaded a multi-page template with **ASP.NET Core 2.x** + **.Net Core Framework** + **Authentication** from [https://aspnetboilerplate.com/Templates](https://aspnetboilerplate.com/Templates)
+
+### Getting Started
+
+There are two Entity Framework Core provider for MySql that are mentioned in Micrososft Docs. One of them is 
+[Official MySQL EF Core Database Provider](https://docs.microsoft.com/en-us/ef/core/providers/mysql/) and 
+other is [Pomelo EF Core Database Provider for MySQL](https://docs.microsoft.com/en-us/ef/core/providers/pomelo/).
+
+> **NOTE:** Official provider doesn't support EF Core 2.0 yet. So we will use Pomelo.
+> 
+> Related issue: https://github.com/aspnet/EntityFrameworkCore/issues/10065#issuecomment-336495475
+
+### Install 
+
+Install [`Pomelo.EntityFrameworkCore.MySql`](https://www.nuget.org/packages/Pomelo.EntityFrameworkCore.MySql/) nuget package to ***.EntityFrameworkCore** project. 
+
+### Configuration
+
+We need some configurations and workarounds to use MySql with ASP.NET Core and Entity Framework Core. 
+There isn't any ABP related configuration. 
```

Comment:
There isn't any ABP related configuration **yet**

Reply:
There is not an ABP related configuration yet.